devops/docker/smqtk_iqr_playground/run_container.gpu.sh

```diff
@@ -15,7 +15,7 @@ IQR_GUI_PORT_PUBLISH=5000
 IQR_REST_PORT_PUBLISH=5001
 
 # Replace SOMETHING_HERE by the real version (probably a date)
-IMAGE_VERSION=SOMETHING_HERE
+IMAGE_VERSION=`docker image ls | grep iqr | awk '{print $2}' | cut -d\- -f1`
```

Comment:
Should possibly add an ``if [ -z "$IMAGE_VERSION"]; then <error or something>; fi`` after this to capture if this returns an empty string because there are no docker images present matching your grep.